src/UIAutomation/FunctionalTests/FunctionalTests.vcxproj

```diff
@@ -93,12 +93,12 @@
       <PrecompiledHeader>Use</PrecompiledHeader>
       <WarningLevel>Level4</WarningLevel>
       <SDLCheck>true</SDLCheck>
-      <AdditionalIncludeDirectories>$(VCInstallDir)UnitTest\include;%(AdditionalIncludeDirectories)</AdditionalIncludeDirectories>
+      <AdditionalIncludeDirectories>$(VCInstallDir)UnitTest\include;$(SolutionDir)UiaOperationAbstraction;%(AdditionalIncludeDirectories)</AdditionalIncludeDirectories>
       <PreprocessorDefinitions>WIN32;_DEBUG;%(PreprocessorDefinitions)</PreprocessorDefinitions>
       <UseFullPaths>true</UseFullPaths>
       <PrecompiledHeaderFile>pch.h</PrecompiledHeaderFile>
       <LanguageStandard>stdcpp17</LanguageStandard>
-      <ConformanceMode>true</ConformanceMode>
+      <ConformanceMode>false</ConformanceMode>

```

Comment:
>false [](start = 23, length = 5)

It's unfortunate this change is needed, since the WinRT API should be `permissive-` conformant.

Regardless, it might make sense to have the abstraction and WinRT APIs separate anyway? They kind of are separate projects too...

Anyway, not a big deal or a blocker in any way for this PR, just thinking outloud.

Reply:
Yeah, the issue I ran into that required this was that UiaBool was being used in the abstraction headers before it was defined. I'm not sure if there was something else that also required conformance mode false, but I didn't dig too deep.